lib/models/subreddit.dart

```diff
@@ -18,6 +18,44 @@ class Subreddit {
 
   final String pictureUrl;
 
-  const Subreddit({Key? key, required this.displayName, required this.fullName, required this.posts, required this.membersCount,
-    required this.description, required this.link, required this.bannerUrl, required this.pictureUrl});
+  final PostSort sortingMethod;
+
+  final PostTopSort? topSortingMethod;
```

Comment:
Why can it be null ? we should initialize it a top of the day or something else but not null